scheme/uploadMock.js

```diff
@@ -0,0 +1,84 @@
+const admin = require("firebase-admin");
+//node_modules/firebase-admin");
+const serviceAccount = require("./secret.json");
+const data = require("./data.json");
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: "https://mutualaid-757f6.firebaseio.com",
```

Comment:
should this be in the .env file?

Reply:
ehm, this is secret from my own user account, and I do not want ppl go around upload things anyway,  the plan is, each week, I will be going and remove old data, upload new data that is correctly format so that we have fresh things to work with